tools/bazel.rc

```diff
@@ -1 +1,6 @@
-build --copt=--std=c++14 -c opt
+build --copt=--std=c++14
+
+build --copt=-march=corei7 --copt=-mtune=core-avx2
```

Comment:
not sure if we want to add these as they won't be true for all computers.

Reply:
corei7 is pretty ancient at this point, requiring only SSE4.2. Intel Core i7 (“Nehalem”), Intel Atom (Silvermont core), AMD Bulldozer, AMD Jaguar, and later processors. Tuning for a later architecture doesn't affect compatibility, just the optimizer's choices.

SSE4.2 includes an alternate instruction encoding for the SSE2 intrinsics, so in theory it might be faster than SSE2 for flat_hash_set and F14 even if they only use SSE2. The most important feature for us is that SSE4.2 has a hardware CRC instruction that we used for bit mixing when we don't trust the hash, but that is not needed for the microbenchmark.

https://gcc.gnu.org/onlinedocs/gcc-4.8.4/gcc/i386-and-x86-64-Options.html